README.md

```diff
@@ -8,11 +8,19 @@ Action types for the redux dispatcher must be unique.  An easy way to get unique
 
 This library ensures that you follow this pattern, producing clean, readable action type namespacing.
 
+The function will throw an error if one of the args are not a string, or if you pass the same string twice. 
+
+For the next major release, the function will expect a string as first argument for the namespace, and and array of strings as second parameter for the types
```

Comment:
foreshadowing an unrelated future feature probably not needed

Reply:
So I'll remove the 'will' from the first sentence since it does it now, and remove the second sentence.